rasterfoundry/models/project.py

```diff
@@ -13,12 +16,49 @@
         TileLayer,
     )
 
+RV_CPU_QUEUE = 'raster-vision-cpu'
+RV_CPU_JOB_DEF = 'raster-vision-cpu'
+DEVELOP_BRANCH = 'develop'
+
+
+def start_raster_vision_job(client, job_name, command, job_queue=RV_CPU_QUEUE,
+                            job_definition=RV_CPU_JOB_DEF,
+                            branch_name=DEVELOP_BRANCH, attempts=1):
+    """Start a raster-vision Batch job.
+
+    Args:
+        client: Boto3 Batch client
+        job_name (str): name of the Batch job
+        command (str): command to run inside the Docker container
+        job_queue (str): name of the Batch job queue to run the job in
+        job_definition (str): name of the Batch job definition
+        branch_name (str): branch of the raster-vision repo to use
+        attempts (int): number of attempts for the Batch job
+
+    Returns:
+        job_id (str): job_id of job started on Batch
+    """
+    # `run_script.sh $branch_name $command` downloads a branch of the
+    # raster-vision repo and then runs the command.
+    job_command = ['run_script.sh', branch_name, command]
+    job_id = client.submit_job(
+        jobName=job_name, jobQueue=job_queue, jobDefinition=job_definition,
+        containerOverrides={
+            'command': job_command
+        },
+        retryStrategy={
+            'attempts': attempts
+        })['jobId']
+
+    return job_id
+
 
 class Project(object):
     """A Raster Foundry project"""
 
     TILE_PATH_TEMPLATE = '/tiles/{id}/{{z}}/{{x}}/{{y}}/'
     EXPORT_TEMPLATE = '/tiles/{project}/export/'
+    batch_client = boto3.client('batch')
```

Comment:
I think we should create the batch client in the function call. Not all Raster Foundry users will have AWS credentials set up or necessarily know about AWS (this is kind of an advanced feature). I don't think this is going to be called enough for a single project that the performance benefit will be noticeable.

This should also fix the build then the client won't be created during tests.

Reply:
Good idea.